setup.cfg

```diff
@@ -11,6 +11,7 @@ classifiers =
     Programming Language :: Python :: 3.8
 description = NeuroImaging Workflows provides processing tools for magnetic resonance images of the brain.
 license = 3-clause BSD
+license_file = LICENSE
```

Comment:
I agree with this change, but I can't see that the LICENSE file is being copied into the sdist. Should we also re-include it in the MANIFEST.in?

EDIT: Actually, we might just need to pin the correct setuptools version [here](https://github.com/nipreps/niworkflows/blob/master/.circleci/config.yml#L359), and [here](https://github.com/nipreps/niworkflows/blob/master/.circleci/config.yml#L384)

--

> In #507, the LICENSE was removed from the sdist, which seems like a mistake to me.

Yup, for some reason I understood the LICENSE file is automatically added to the sdist without need of specifying it in the MANIFEST.in file. Then I failed to double-check that.

Reply:
This got me thinking further. When we build the package on CircleCI for distribution, we should use the most current stable version of setuptools, to maximize the support. That doesn't mean we should do the same for the distribution tests, there we may want to play with the different versions of setuptools (maybe for `pip` and `python setup.py` installations directly from the source).

Reply:
Okay this was updated in 66487e662ce0817bd694a1686c876945cd566e43